### ISSUE TITLE
[intersphinx] Sphinx docs objects.inv broken (probably JavascriptDomain bug)

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#473]
With Sphinx 1.0 the objects.inv of the Sphinx docs is broken, causing the client to choke at parsing:

```
WARNING: intersphinx inventory 'http://sphinx.pocoo.org/objects.inv' not readable due to ValueError: unknown or unsupported inventory version
```

The offending line is no. 369:
    u' js:class 1 domains.html#$ -'}}}
    {{{u' js:class 1 domains.html#$ -'
This line is missing a symbol name, which causes the inventory parser to fail when splitting the line into components.  The exception is shadowed by the catch-all logic in intersphinx, but the original traceback is:

```
Traceback (most recent call last):
  File "/home/lunar/devel/projects/sphinx-contrib/ansi/.tox/doc/lib/python2.6/site-packages/sphinx/ext/intersphinx.py", line 126, in fetch_inventory
    invdata = read_inventory_v2(f, uri, join)
  File "/home/lunar/devel/projects/sphinx-contrib/ansi/.tox/doc/lib/python2.6/site-packages/sphinx/ext/intersphinx.py", line 94, in read_inventory_v2
    name, type, prio, location, dispname = line.rstrip().split(None, 4)
ValueError: need more than 4 values to unpack
```

The offending "domains.rst" however looks ok, so I assume, that this is a bug in the javascript domain.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/473
- Originally reported by: Sebastian Wiesner
- Originally created at: 2010-07-26T22:43:59.701

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-07-27 12:29:22+00:00_

Should be fixed in <<cset 954e8d0220866151ca2e57cf90885c008fc4ae35>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-08-14 15:19:33+00:00_

Removing version: 1.0beta (automated comment)
